beacon-chain/core/state/validity_conditions.go

```diff
@@ -0,0 +1,60 @@
+package state
```

Comment:
Validity conditions will be moved here in order to prevent import cycles and keep types/block.go clean